microsoft-edge/devtools-guide-chromium/css/index.md

```diff
@@ -74,7 +74,7 @@ Use the **Styles** panel when you want to change or add CSS declarations to an e
 
     The `background-color:honeydew` declaration is applied to the element using the `element.style` section of the **Styles** panel:
 
-   ![Add a CSS declaration to the element using the Styles panel](./index-images/css-elements-add-background-color-to-me-styles-p.png)
+   ![Add a CSS declaration to the element using the Styles panel](./index-images/add-background-color-to-me-styles-p.png)
```

Comment:
```suggestion
   ![Add a CSS declaration to the element by using the Styles panel](./index-images/add-background-color-to-me-styles-p.png)
```
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/u/using-by-using-with - seems to lean toward "by using", which I would use to firm up the parsing for the reader.